src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -3217,6 +3217,7 @@ class Template(Model):
     _attribute_map = {
         'revision_suffix': {'key': 'revisionSuffix', 'type': 'str'},
         'containers': {'key': 'containers', 'type': '[Container]'},
+        'initContainers': {'key': 'initContainers', 'type': '[Container]'},
```

Comment:
In the spec, it looks like initContainers are different from app containers because [app containers have probes](https://github.com/Azure/azure-rest-api-specs/blob/9f33c53e4b6d09e22329df29ac2ef404fe2149de/specification/app/resource-manager/Microsoft.App/stable/2022-10-01/CommonDefinitions.json#L115). Would this be a problem here?

Reply:
@anthonychu if we set probes for a InitContainer, it will not be processed. We handle it in the actual API create/update API.